hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/utils/columnSearch.tsx

```diff
@@ -69,8 +69,16 @@ class ColumnSearch extends React.PureComponent {
     filterIcon: (filtered: boolean) => (
       <Icon type='search' style={{color: filtered ? '#1890ff' : undefined}}/>
     ),
-    onFilter: (value: string, record: any) =>
-      record[dataIndex].toString().toLowerCase().includes(value.toLowerCase()),
+    onFilter: (value: string, record: any) => {
+      if (record[dataIndex] !== undefined || record[dataIndex] !== null) {
+          return typeof (record[dataIndex]) === typeof {}
+            ? Boolean (record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase())) ? true :false)
+            : record[dataIndex].toString().toLowerCase().includes(value.toLowerCase())
+        }
+        else {
```

Comment:
Is this else needed ?

Reply:
@devmadhuu 
Boolean (record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase())))
Added short code for array search.

Reply:
I guess it is safer to have it in place, in case there is a scenario like the data-fetching for say DataNodes failed, so now record[DataIndex] is undefined. In that case if we try to perform a search in the column, it might give error.